awx/ui_next/src/screens/Schedule/ScheduleList/ScheduleListItem.jsx

```diff
@@ -0,0 +1,157 @@
+import React from 'react';
+import { bool, func } from 'prop-types';
+import { withI18n } from '@lingui/react';
+import { t } from '@lingui/macro';
+import { Link } from 'react-router-dom';
+import {
+  DataListItem,
+  DataListItemRow,
+  DataListItemCells as _DataListItemCells,
+  Tooltip,
+} from '@patternfly/react-core';
+import { PencilAltIcon } from '@patternfly/react-icons';
+
+import ActionButtonCell from '@components/ActionButtonCell';
+import { DetailList, Detail } from '@components/DetailList';
+import DataListCell from '@components/DataListCell';
+import DataListCheck from '@components/DataListCheck';
+import ListActionButton from '@components/ListActionButton';
+import Switch from '@components/Switch';
+import VerticalSeparator from '@components/VerticalSeparator';
+import styled from 'styled-components';
+import { Schedule } from '@types';
+import { formatDateString } from '@util/dates';
+
+const DataListItemCells = styled(_DataListItemCells)`
+  ${DataListCell}:first-child {
+    flex-grow: 2;
+  }
+`;
+
+function ScheduleListItem({
+  i18n,
+  isSelected,
+  onSelect,
+  onToggleSchedule,
+  schedule,
+  toggleLoading,
+}) {
+  const labelId = `check-action-${schedule.id}`;
+
+  const jobTypeLabels = {
+    inventory_update: i18n._(t`Inventory Sync`),
+    job: i18n._(t`Playbook Run`),
+    project_update: i18n._(t`SCM Update`),
+    system_job: i18n._(t`Management Job`),
+    workflow_job: i18n._(t`Workflow Job`),
+  };
+
+  let scheduleBaseUrl;
+
+  switch (schedule.summary_fields.unified_job_template.unified_job_type) {
```

Comment:
I'd be inclined to do some object destructuring for these paths just to make them a bit easier to read.